app.R

```diff
@@ -78,18 +77,21 @@ ui <- fluidPage(
                                         "Sassafras","Tilia","Ulmus","Viburnum"),
                            inline = FALSE),
       ),
-      plotOutput("avg_dbh")
+      mainPanel(
+        plotOutput("distPlot")
+      )
     )
 )
```

Comment:
It seems there is some mis-ordered and mis-placed parentheses here. It might be a good idea to take the original `app.R` file [here](https://github.com/390-2020-09-Fall/shiny_app/blob/bb5b354cda607c71eb4705b0f198eadd157051bd/app.R) and start over. This might be easier than a wild goose chase of parentheses